src/components/picker/index.tsx

```diff
@@ -196,6 +224,27 @@ const Picker = (props: PropsWithChildren<PickerProps> & ForwardRefInjectedProps
     useSafeArea
   ]);
 
+  const renderPickerInnerInput = () => {
+    if (fieldType === PickerFieldTypes.filter) {
+      return (
+        <Text text70 style={others.style}>
```

Comment:
Add the `$textDefault` modifier for the dark mode support

Reply:
same.. 
At this point, it just proves my first answer (: